app/grants/urls.py

```diff
@@ -31,8 +31,8 @@
 app_name = 'grants'
 urlpatterns = [
     path('', grants, name='grants'),
-    path('clr/<slug:round_num>', clr_grants, name='clr_grants'),
-    path('clr/<slug:round_num>/', clr_grants, name='clr_grants'),
+    path('clr/<int:round_num>', clr_grants, name='clr_grants'),
+    path('clr/<int:round_num>/<str:sub_round_slug>', clr_grants, name='clr_grants'),
```

Comment:
why would you have round num and slug ?
Is this to group all the clr for round 8 under clr/8/...?

For custom CLR like matic / zcash 
I'm not sure if this makes sense 

I'd lean towards 
- keeping how it was 
- keeping both

Reply:
round num = 7
slug = community, infra tech, or dapp tech

there can be multiple slugs in the same round.